src/node/BrowserFetcher.ts

```diff
@@ -561,7 +562,7 @@ function httpRequest(
 
   type Options = Partial<URL.UrlWithStringQuery> & {
     method?: string;
-    agent?: ProxyAgent;
+    agent?: HttpsProxyAgent | SocksProxyAgent | any;
```

Comment:
Docs explicitly said no `any` :) But I couldnt found better way to handle those different types at once. Some pointers really appreciated :)

Reply:
I think in this case `| any` is not really needed.